drag_and_drop_v2/templates/html/drag_and_drop_edit.html

```diff
@@ -116,6 +116,7 @@
                 <h3>{% trans "Items" %}</h3>
             </header>
             <section class="tab-content">
+                <div class="xblock-editor-error-messages" aria-live="polite" tabindex="0"></div>
```

Comment:
I definitely want to avoid re-working the XBlock editor substantially for this PR if we can help it.

If the UX for this particular validation is a problem, I'd rather leave out this warning. Besides, I think there's a bug in the current logic: if I have 10 items and two zones and I limit each zone to 5 items but allow each item to be placed in any zone, then it will raise an error, though that's a perfectly valid problem. Validating this in a way that accounts for such problems is likely going to be overly complex.